File: codificacao-c#/URI1035.cs
using System;

class URI
{

    static void Main(string[] args)
    {
        string[] linha = Console.ReadLine().Split(' ');
        int A = Convert.ToInt32(linha[0]);
        int B = Convert.ToInt32(linha[1]);
        int C = Convert.ToInt32(linha[2]);
        int D = Convert.ToInt32(linha[3]);
        bool res = true;
        if (B <= C)
        {
            res = false;
        }
        if (D <= A)
        {
            res = false;
        }
        if ((C + D) <= (A + B))
        {
            res = false;
        }
        if (C < 0 || D < 0)
        {
            res = false;
        }
        if ((A % 2) != 0)
        {
            res = false;
        }
        if (res)
        {
            Console.WriteLine("Valores aceitos");
        }
        else
        {
            Console.WriteLine("Valores nao aceitos");
        }

    }

}

File: codificacao-java/Uri1035.java
import java.io.IOException;
import java.util.Scanner; 

public class Main {
 
    public static void main(String[] args) throws IOException {
		Scanner sc = new Scanner(System.in);
		int A = sc.nextInt();
		int B = sc.nextInt();
		int C = sc.nextInt();
		int D = sc.nextInt();
		boolean res = true;
		if(B <= C)
		{
			res = false;
		}
		if(D <= A)
		{		
			res = false;
		}
		if((C + D) <= (A + B))
		{
			res = false;
		}
		if(C < 0 || D < 0)
		{
			res = false;
		}
		if((A % 2) != 0)
		{
			res = false;
		}
		if(res)
		{
			System.out.println("Valores aceitos");
		}
		else
		{
			System.out.println("Valores nao aceitos");
		}
    }
 
}